docs/website/docs/dlt-ecosystem/destinations/athena.md

```diff
@@ -137,6 +137,13 @@ force_iceberg = "True"
 
 For every table created as an iceberg table, the Athena destination will create a regular Athena table in the staging dataset of both the filesystem and the Athena glue catalog, and then copy all data into the final iceberg table that lives with the non-iceberg tables in the same dataset on both the filesystem and the glue catalog. Switching from iceberg to regular table or vice versa is not supported.
 
+#### `merge` support
+The `merge` write disposition is supported for Athena when using iceberg tables.
+
+> Note that:
+> 1. there is a risk of tables ending up in inconsistent state in case a pipeline run fails mid flight, because Athena doesn't support transactions, and `dlt` uses multiple DELETE/UPDATE/INSERT statements to implement `merge`, 
```

Comment:
Why not use the `merge` syntax that is available in Athena?

Reply:
Full discussion is here: https://github.com/dlt-hub/dlt/pull/1294.

_TLDR_: we're planning to add a new merge strategy based on the MERGE statement, but this isn't properly specced yet. We want consistent behavior accross different destinations, hence the implementation of `delete-insert` for `athena` now. MERGE-based `merge` for `athena` will come together with the other destinations.